examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/data-assets/graph_asset_decorator.py

```diff
@@ -0,0 +1,17 @@
+import dagster as dg
+
+
+@dg.op
+def step_one(): ...
+
+
+@dg.op
+def step_two(): ...
```

Comment:
What do you think of adding a RetryPolicy to the ops just to give users a small taste of what ops can offer?

Reply:
I like the idea